package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -601,7 +683,8 @@ def run(self, **kwargs):
             if not selection_resn_id:
                 self._timeseries.append([])
                 continue
-            selection_resn_id = ['(resname {} and resid {})'.format(resname, resid) for resname, resid in selection_resn_id]
+            selection_resn_id = ['(resname {} and resid {})'.format(resname,
+            resid) for resname, resid in selection_resn_id]
```

Comment:
Please indent for legibility. Follow [PEP8 indentation rules](https://www.python.org/dev/peps/pep-0008/#indentation)